web/packages/teleport/src/components/ButtonLockedFeature/ButtonLockedFeature.tsx

```diff
@@ -29,17 +30,24 @@ export type Props = {
   [index: string]: any;
 };
 
-const salesUrl = 'https://goteleport.com/signup/enterprise/';
+const salesUrl = 'https://goteleport.com/r/upgrade-team';
 
 export function ButtonLockedFeature({
   children,
   noIcon = false,
   event,
   ...rest
 }: Props) {
+  const ctx = useTeleport();
+  const version = ctx.storeUser.state.cluster.authVersion;
+  const isEnterprise = ctx.isEnterprise;
+
   function handleClick() {
     userEventService.captureCtaEvent(event);
-    window.open(salesUrl, 'blank');
+    window.open(
+      `${salesUrl}?${getParams(version, isEnterprise, event)}`,
+      'blank'
+    );
```

Comment:
afaik, we don't use `window.open` for a button click out, we instead do something like this: https://github.com/gravitational/teleport/blob/42c95bfd6b70484936fccd8432df5f570c41ef78/web/packages/teleport/src/Discover/Shared/ActionButtons.tsx#L47

Reply:
Thanks, updated in https://github.com/gravitational/teleport/pull/26086/commits/c67b9bbb4121df7e378271ca2728ade32d4a2469.